packages/atomic/src/utils/initialization-utils.tsx

```diff
@@ -290,4 +290,26 @@ export function DeferUntilRender() {
   };
 }
 
+export function RemainHiddenUntilDescendantsRendered() {
```

Comment:
The reason was primarily because I expected some other components may need it too in the future, but now I think i prefer moving this inside the component. We can make it a decorator later if we need to re-use it.